projects/packages/calypsoify/src/class-jetpack-calypsoify.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * This is Calypso skin of the wp-admin interface that is conditionally triggered via the ?calypsoify=1 param.
+ *
+ * @package automattic/jetpack-calypsoify
+ */
+
+namespace Automattic\Jetpack\Calypsoify;
+
+use Automattic\Jetpack\Status;
+
+/**
+ * Class Jetpack_Calypsoify
+ */
+class Jetpack_Calypsoify {
+
+	const PACKAGE_VERSION = '0.1.0-alpha';
+
+	/**
+	 * Singleton instance of `Jetpack_Calypsoify`.
+	 *
+	 * @var object
+	 */
+	public static $instance = false;
+
+	/**
+	 * Is Calypsoify enabled, based on any value of `calypsoify` user meta.
+	 *
+	 * @var bool
+	 */
+	public $is_calypsoify_enabled = false;
+
+	/**
+	 * Jetpack_Calypsoify constructor.
+	 */
+	private function __construct() {
+		add_action( 'admin_init', array( $this, 'setup' ), 4 );
+	}
+
+	/**
+	 * Singleton.
+	 *
+	 * @return Jetpack_Calypsoify
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Setup function that is loaded on the `wp_loaded` hook via the constructor.
+	 */
+	public function setup() {
+		$this->is_calypsoify_enabled = isset( $_GET['calypsoify'] ) && 1 === (int) $_GET['calypsoify'] && $this->is_page_gutenberg(); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+
+		$this->check_meta();
+
+		if ( $this->is_calypsoify_enabled ) {
+			add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_for_gutenberg' ), 100 );
+		}
+	}
+
+	/**
+	 * Enqueues scripts, data, and styles for Gutenberg.
+	 */
+	public function enqueue_for_gutenberg() {
+		$site_suffix = ( new Status() )->get_site_suffix();
+		wp_enqueue_style( 'calypsoify_wpadminmods_css', plugin_dir_url( __FILE__ ) . 'style-gutenberg.min.css', false, self::PACKAGE_VERSION );
```

Comment:
Since we changed the file structure with css files currently living under the `css` folder we'll need to fix the path here as well.
I guess the `style-gutenberg.min.css` will be created on build but added to the `.gitignore` too?

Reply:
Good point, thanks!